client/lib/parse-address.js

```diff
@@ -0,0 +1,4 @@
+export default function parseAddress(address) {
+  const parsedAddress = address.replaceAll('< span class="street-address" >', '').replaceAll('<span class="street-address">', '').replaceAll('<span class="locality">', '').replaceAll('<span class="region">', '').replaceAll('<span class="postal-code">', '').replaceAll('<span class="country-name">', '').replaceAll('</span>', '');
```

Comment:
if i understand correctly, you are trying to remove all spans from the address? The code is a little tough to read when it stretches this far past 80 columns. Try some indentation.

```js
function parseAddress(address) {
  const parsedAddress = address
    .replaceAll('< span class="street-address" >', '')
    .replaceAll('<span class="street-address">', '')
    .replaceAll('<span class="locality">', '')
    .replaceAll('<span class="region">', '')
    .replaceAll('<span class="postal-code">', '')
    .replaceAll('<span class="country-name">', '').replaceAll('</span>', '');
  return parsedAddress;
}
```